pwndbg/__init__.py

```diff
@@ -81,6 +87,10 @@
 
 comments.init()
 
-from pwndbg.gdblib import config_mod
+from pwndbg.gdblib import config_mod as gdblib_config_mod
+
+gdblib_config_mod.init_params()
+
+from pwndbg.gdblib.prompt import show_hint
 
-config_mod.init_params()
+show_hint()
```

Comment:
I think you can remove these lines and leave them in the other PR. They seem to be the ones breaking the unit test anyway.

Reply:
This has a lot to do with the order of imports sometimes causing `$ida` not to show up on the startup prompt. I did originally intend to have this change be present in only one of the PRs, but my local tests started failing for that when I removed it.

Maybe something changed when I merged the upstream changes that fixed it, but, I don't think it hurts to leave them here, just in case.